Assets/Scripts/Actors/BuddyStats.cs

```diff
@@ -3,18 +3,32 @@
 
 public class BuddyStats : MonoBehaviour
 {
-	[SerializeField] Stat statType = Stat.Invalid;
+	public Stat statType = Stat.Invalid;
 	[SerializeField] bool canDecreaseStamina = true;
 	[SerializeField] float decreaseResourcesTime = 60.0f; //seconds
 	[SerializeField] float currResourceTimer = 0.0f;
 	[SerializeField] int startingApples = 10;
 	[SerializeField] int apples = 0;
-
-	public GodTag owner = null;
+	[SerializeField] GodTag _startingOwner = null;
 
 	PlayerActorStats _actorStats = null;
 
-	public Material heartMaterial;
+	[SerializeField] Material _heartMaterial;
+	[SerializeField] Material _sadMaterial;
+
+	private GodTag _owner = null;
+	public GodTag owner
+	{
+		get { return _owner; }
+		set
+		{
+			_owner = value;
+			if ( _owner )
+			{
+				_actorStats = _owner.gameObject.GetComponent<PlayerActorStats>();
+			}
+		}
+	}
```

Comment:
Could we just replace this with PlayerActorStats since the tag aspect is only being used to get PlayerActorStats?